### PR TITLE
Fat 2478 c343349

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://foliotest.testrail.io/index.php?/cases/view/343349
***
https://issues.folio.org/browse/FAT-2478
***

### CODE REVIEW

File: cypress/e2e/inventory/search/browse-call-numbers.cy.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import TestTypes from '../../../support/dictionary/testTypes';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import InventoryActions from '../../../support/fragments/inventory/inventoryActions';
+import devTeams from '../../../support/dictionary/devTeams';
+
+describe('ui-inventory: browse call numbers', () => {
+  beforeEach('navigate to inventory', () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
```
Comment (IhorBohdan) [inline cypress/e2e/inventory/search/browse-call-numbers.cy.js:9]: use the method cy.loginAsAdmin

File: cypress/e2e/inventory/single-record-import/import-by-oclc-source-folio.cy.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+};
+
+describe('ui-inventory: import by OCLC', () => {
+  before('create test data', () => {
+    cy.getAdminToken()
+      .then(() => {
+        Z3950TargetProfiles.changeOclcWorldCatValueViaApi('100473910/PAOLF');
+        InventorySearchAndFilter.createInstanceViaApi()
+          .then(({ instanceData }) => {
+            console.log(instanceData);
```
Comment (IhorBohdan) [inline cypress/e2e/inventory/single-record-import/import-by-oclc-source-folio.cy.js:34]: Delete console.log

File: cypress/support/fragments/inventory/inventoryInstance.js [hunk 3]
```diff
@@ -67,27 +73,62 @@ const validOCLC = { id:'176116217',
...
+const verifyInstancePublisher = (indexRow, indexColumn, type) => {
+  cy.expect(descriptiveDataAccordion.find(MultiColumnList({ id: 'list-publication' }))
+    .find(MultiColumnListRow({ index: indexRow })).find(MultiColumnListCell({ columnIndex: indexColumn }))
+    .has({ content: type }));
+};
+
+const verifyInstanceSubject = (indexRow, indexColumn, value) => {
+  cy.expect(Accordion('Subject')
+    .find(MultiColumnList({ id: 'list-subject' }))
+    .find(MultiColumnListRow({ index: indexRow })).find(MultiColumnListCell({ columnIndex: indexColumn }))
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/inventoryInstance.js:101]: move to const

File: cypress/e2e/inventory/search/browse-call-numbers.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/inventory/search/browse-call-numbers.cy.js:9]: Done

File: cypress/e2e/inventory/single-record-import/import-by-oclc-source-folio.cy.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/e2e/inventory/single-record-import/import-by-oclc-source-folio.cy.js:34]: Done

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**